Assets/MRTK/SDK/Editor/Inspectors/UX/StateVisualizer/StateVisualizerInspector.cs

```diff
@@ -0,0 +1,402 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+using Microsoft.MixedReality.Toolkit.UI.Interaction;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
```

Comment:
Make sure that all experimental classes are in a separate experimental namespace, following the [guide here](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-code-should-be-in-a-special-namespace)